.markdownlint-cli2.cjs

```diff
@@ -3,6 +3,27 @@ const gitDiff = require('./packages/markdownlint-rule-mui/git-diff');
 const tableAlignment = require('./packages/markdownlint-rule-mui/table-alignment');
 const terminalLanguage = require('./packages/markdownlint-rule-mui/terminal-language');
 const duplicateH1 = require('./packages/markdownlint-rule-mui/duplicate-h1');
+const searchReplace = require('markdownlint-rule-search-replace');
+
+const brandNames = [
+  { search: 'Material UI', replace: 'Material&nbsp;UI' },
+  { search: 'MUI X', replace: 'MUI&nbsp;X' },
+  { search: 'Base UI', replace: 'Base&nbsp;UI' },
+  { search: 'MUI System', replace: 'MUI&nbsp;System' },
+  { search: 'MUI Store', replace: 'MUI&nbsp;Store' },
+  { search: 'MUI Core', replace: 'MUI&nbsp;Core' },
+  { search: 'MUI Toolpad', replace: 'Toolpad' },
+  { search: 'MUI Toolpad', replace: 'Toolpad' },
```

Comment:
Are these duplicated or is there a difference between them?

Reply:
One of them has non-breaking space 🙂